test/TestUtil.ts

```
@@ -111,18 +102,26 @@
 
   static async getTableNames() {
     if (!this.tables) {
-      const database = this.getDatabase();
-      const sql = `SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = '${database}';`;
-      const rows = await this.query(sql);
-      this.tables = rows.map(row => row.TABLE_NAME);
+      const config = this.getDatabaseConfig();
+      if (config.type === DATABASE_TYPE.MySQL) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)